### ISSUE TITLE
Implement complex attribute types

### ISSUE THREAD

**Issue by jonkoops** [keycloak/keycloak-ui#3473]
### Description

Currently we have a basic implementation for attributes that supports string based values (as pictured below).

![A screenshot of the Attributes tab of a user](https://user-images.githubusercontent.com/695720/193823092-84c0c798-07c0-439e-b3b9-32beb892580f.png)

However, the value in this form can be any valid [JSON value](https://www.json.org/json-en.html) data type. The UI should be designed to accommodate these complex values.

**Comment by xianli123** [comment]
Hi @jonkoops I've checked these 2 issues. Does it mean that the UI should support multiple values for the same key and values in different formats that are listed in the [JSON value](https://www.json.org/json-en.html)?

**Comment by jonkoops** [comment]
Yes, we'd at least need a way to provide multiple string values (an array). As for the other data-types, that would require some research to see if they are accepted, but it would be good to keep it into account in the design.

**Comment by xianli123** [comment]
Here is the updated design for your reference. https://www.sketch.com/s/197538a9-58a1-4da2-8c9b-77f20a155b81/v/R4mME8/prototype/a/37965EEF-674A-41A0-97B8-B2788FEF6AA8/
